setup.py

```diff
@@ -221,16 +231,19 @@ def get_requirements(requirement_files: Union[str, List[str]]) -> List[str]:
 
     requirements: List[str] = []
     for requirement_file in requirement_files:
-        with open(f"requirements/{requirement_file}.txt", "r", encoding="utf8") as file:
+        with open(f"requirements/{requirement_file}.txt", "r", encoding="UTF-8") as file:
             for line in file:
                 package = line.strip()
                 if package and not package.startswith(("#", "-f")):
+                    requirement = Requirement.parse(package)
+                    if requirement.name in ("torch", "torchvision"):
+                        package = update_torch_requirement(requirement)
                     requirements.append(package)
 
     return requirements
 
 
-REQUIRED_PACKAGES = get_requirements(requirement_files=["base", "dev", "openvino"])
+REQUIRED_PACKAGES = get_requirements(requirement_files=["base", "dev", "openvino", "detection"])
```

Comment:
I modified this with the goal of enabling detection training to work immediately when otx is installed.
Unfortunately, mmdetection requires installation in an environment with torch installed.
So, I want to guide users to install torch for each environment and install otx to make it work.
(torch install -> otx install,  it working now)

Reply:
It seems to work well locally, but it seems that the environment is not properly installed in the tox test that works in github action. 

@ashwinvaidya17 Could you give me some advice? I'm not familiar with the tox. It looks like mmcv is installed without being properly built.. 

test works fine with `otx/algorithms/detection/init_venv.sh` env

Reply:
I'll have a look

Reply:
@harimkang is there a reason that torch version is limited 1.9.0 in mmdet https://github.com/openvinotoolkit/mmdetection/blob/ote/requirements/build.txt#L4? This ends up overwriting the torch version on ote and leads to mmcv-full installation removing the old torch version using which mmcv is build. It leads to `libtorch_cuda_cu.so: cannot open shared object file: No such file or directory`

Reply:
Thanks for the detailed analysis! 
Perhaps there is no other reason and the our mmdetection version is a bit lower than expected..  We plan to change it to public mmdet later and update the version, but not right now.  Tomorrow I will check if I can control the mmcv version and the torch version of mmdet.

However, if this is difficult, first leave only pre-commit, and in the github action, the detection test should be omitted and put again next time..

Reply:
But I don't think this is the only issue here. I set the version to 1.9.0 in base.txt and I am still getting some errors. I am still investigating the issue.

Reply:
First of all, it has been modified so that the detection test is not run in tox. I'll add it later when the installation is properly organized..